infra/resources/_modules/apim/apim.tf

```diff
@@ -0,0 +1,37 @@
+module "apim_v2_wallet_api" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//api_management_api?ref=v8.12.2"
+
+  name                  = format("%s-ioweb-wallet", var.project_legacy)
```

Comment:
```suggestion
  name                  = "IO Web - Wallet"
```

Reply:
i've seen that in other configurations (E.g. io-infra) is used the `format("%s-NAME", product)` form, maybe is better to maintain the seme format.